.github/ISSUE_TEMPLATE/new-session.yaml

```
@@ -0,0 +1,101 @@
+name: 🤝 Organize a new Schrödinger Hat session
+description: Create a cell in your city by organizing a Schrödinger Hat Session
```

```suggestion
description: Organize a Schrödinger Session in your city
```

I don't like here the word `cell`